edx_reverification_block/xblock/reverification_block.py

```diff
@@ -48,6 +48,13 @@ class ReverificationBlock(XBlock):
         help="Reverification XBlock is configured or not."
     )
 
+    end = DateTime(
```

Comment:
As we discussed, this field needs to be called `due` in order to inherit from its parent XBlock.